rcp/org.testeditor.rcp4.views.teststepselection/src/org/testeditor/rcp4/views/teststepselector/TestStepSelector.xtend

```diff
@@ -93,37 +113,49 @@ class TestStepSelector {
 
 	@Inject
 	@Optional
-	def void updateModelAndView(@EventTopic(SELECTOR_UPDATE_MODEL_AND_VIEW) Object data) {
-		logger.debug("updateModelAndView")
-		val amlInjector = amlInjectorProvider.get
-		models = amlInjector.getInstance(AmlModelsProvider).amlModels
-		Display.getDefault.syncExec[internalUpdateView]
+	def void updateView(@EventTopic(SELECTOR_UPDATE_VIEW) Object data) {
```

Comment:
Really? XtextEditor is more concrete than Object. Wouldn't Object need more implementations? I'm probably missing something, since I don't see any relation to implemented drag and drop editors at this point.